app/src/main/java/com/codesoom/assignment/auth/ClaimTokenAuth.java

```diff
@@ -0,0 +1,28 @@
+package com.codesoom.assignment.auth;
+
+/**
+ * Claim Token 기반 유효한 토큰을 생성하고, 토큰의 유효성을 검증한다
+ * <p>
+ * All Known Extending Classes:
+ * JwtAuth
```

Comment:
```suggestion
 * @see JwtAuth
```
이렇게 바꿔보세요! 컨트롤 클릭을 하면 해당 객체로 가기까지해요!

Reply:
그렇군요! 꿀팁 바로 적용해보겠습니다~